frontend/App.svelte

```diff
@@ -0,0 +1,28 @@
+<script>
+  let stats;
+
+  import("../pkg").then(({ Stats, decrease_stats }) => {
+    stats = Stats.new();
+
+    let start = null;
+
+    const handleStats = timestamp => {
+      const elapsedTime = timestamp - start;
+      start = timestamp;
+      decrease_stats(stats, elapsedTime / 1000);
+      stats = stats; // pffffff
```

Comment:
Svelte's neatness? Maybe rewriting as:
```
stats = decrease_stats(stats, elapsedTime / 1000);
```
would make it less weird

Reply:
Unfortunately this won't work because `decrease_stats` doesn't return the stats, since it's a borrowed reference and AFAIK it's not possible at the moment to do that with `wasm-bindgen`.
Anyway, this was just to try out wasm, I'm not gonna move forward with this at least for a few months (too much unsupported stuff). Thanks for your comments though!